tck/src/main/java/org/eclipse/configjsr/CDIPlainInjectionTest.java

```diff
@@ -189,21 +203,14 @@ private void clear_all_property_values() {
         @ConfigProperty(name="my.double.property")
         private double doubleProperty;
 
-        @Inject
```

Comment:
yikes, seems you killed our new features which are not yet present in MicroProfile. 
That needs a way tighter manual apply of changes I fear.